torch/nn/functional.py

```diff
@@ -406,6 +407,28 @@ def dropout(input, p=0.5, training=False, inplace=False):
     return _functions.dropout.Dropout(p, training, inplace)(input)
 
 
+def alpha_dropout(input, p=0.5, training=False):
+    if p < 0 or p > 1:
+        raise ValueError("dropout probability has to be between 0 and 1, "
+                         "but got {}".format(p))
+
+    if p == 0 or not training:
+        return input
+
+    alpha = -1.7580993408473766
```

Comment:
Thanks for the link. For consistency, wouldn't it be better to  set `alpha = - SELU.alpha * SELU.scale' (I see in #1769 that the corresponding values are fixed in class `SELU`) ? The day somebody generalizes class `SELU` at least they will notice that `alpha_dropout()` needs to be adapted.